packages/cli/src/tokens/design-tokens/default/semantic/style.json

```diff
@@ -72,49 +72,17 @@
         }
       }
     },
-    "ingress": {
-      "lg": {
-        "$type": "typography",
-        "$value": {
-          "fontFamily": "{font.family}",
-          "fontWeight": "{font-weight.regular}",
-          "lineHeight": "{line-height.lg}",
-          "fontSize": "{font-size.8}",
-          "letterSpacing": "{letter-spacing.8}"
-        }
-      },
-      "md": {
+    "body": {
+      "xl": {
         "$type": "typography",
         "$value": {
           "fontFamily": "{font.family}",
           "fontWeight": "{font-weight.regular}",
-          "lineHeight": "{line-height.lg}",
+          "lineHeight": "{line-height.md}",
           "fontSize": "{font-size.7}",
           "letterSpacing": "{letter-spacing.8}"
         }
       },
-      "sm": {
-        "$type": "typography",
-        "$value": {
-          "fontFamily": "{font.family}",
-          "fontWeight": "{font-weight.regular}",
-          "lineHeight": "{line-height.lg}",
-          "fontSize": "{font-size.6}",
-          "letterSpacing": "{letter-spacing.7}"
-        }
-      },
-      "xs": {
-        "$type": "typography",
-        "$value": {
-          "fontFamily": "{font.family}",
-          "fontWeight": "{font-weight.regular}",
-          "lineHeight": "{line-height.lg}",
-          "fontSize": "{font-size.5}",
-          "letterSpacing": "{letter-spacing.6}"
-        }
-      }
-    },
```

Comment:
Is this supposed to be removed?

Reply:
The github diff is a bit misleading without the context around it. It's just `ingress` as a whole which has been removed, while `body` still has `md`, `sm` and `xs` (plus the new `xl`). See:
<img width="584" alt="image" src="https://github.com/user-attachments/assets/169384ac-539e-4a0a-a114-5f4d212ef761">